changelog/db.changelog-1.19.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+  <changeSet author="John Watson" id="1">
+    <tagDatabase tag="v1.19.0" />
+  </changeSet>
+  <changeSet author="John Watson" id="2">
+    <update tableName="schemes">
+      <column name="name" value="CS" />
+      <where>"schemeId" = 5</where>
```

Comment:
Why does CS have a 'where' clause while the other two (BPS/FDMR) don't?

Reply:
Scheme 5 was already in there as LNR which isn't needed anymore.  So can update that to CS but for others needed to add fresh.

Reply:
Ah okay, so it was just leftover code from LNR?

Reply:
Yes, we did intend for LNR to be scheme 5 so added it everywhere.  LNR is now being delivered as part of CS, so we're now replacing all usage of 5 to be CS instead of LNR